pkgs/default.nix

```diff
@@ -56684,232 +56598,26 @@ doCheck = false;
 description = "A Haskell-only implementation of the MD5 digest (hash) algorithm";
 license = stdenv.lib.licenses.bsd3;
 
-}) {};
-"purescript" = callPackage
```

Comment:
This shouldn't have gone away, it's causing evaluation errors. How did you regenerate the package set?